crates/matrix-sdk/src/sliding_sync/list/mod.rs

```diff
@@ -1466,7 +1173,8 @@ mod tests {
             }
         };
 
-        list.set_ranges(&[3..=7]).unwrap();
+        list.set_sync_mode(SlidingSyncSelectiveModeBuilder::new().add_range(3..=7).build())
```

Comment:
Just a test:

```suggestion
        list.set_sync_mode(SlidingSyncMode::new_selective().add_range(3..=7))
```

See, it's shorter and less verbose.